libs/ui/src/lib/activities-table/activities-table.component.html

```diff
@@ -53,6 +53,14 @@
       <ion-icon class="mr-2" name="calendar-clear-outline"></ion-icon>
       <span i18n>Export Drafts as ICS</span>
     </button>
+    <button
+      class="align-items-center d-flex"
+      mat-menu-item
+      (click)="onDeleteAllActivities()"
+    >
+      <ion-icon class="mr-2" name="trash-outline"></ion-icon>
+      <span i18n>Delete all activities</span>
```

Comment:
Delete all **A**ctivities

Reply:
actually in English we don't capitalize names like in German :) but i'll change it :P